technical-docs/designs/repos-merge.md

```diff
@@ -0,0 +1,285 @@
+# Moving SPK to Bedrock Repo: A Phased Approach
+
+## Phase 1: Make Changes to Bedrock and SPK Repos
+
+### Changes to Bedrock Repo
+
+By making following changes in existing Bedrock repo minimizes the number of
+changes after the merge and limits the testing scope.
+
+#### Current Bedrock Repo
+
+```
+├── .github
+├── cluster
+├── docs
+├── gitops
+├── pipelines
+├── test
+├── tools
+├── LICENSE
+├── README.md
+├── .gitignore
+├── azure-pipelines.yml
+```
+
+#### Proposed Changes
+
+- Create `src` folder in `cluster` folder
+- Move all files and folders except `src` from `cluster` to `cluster\src`
+- Move `azure-pipelines.yml` file from the root to `cluster\build\pipelines`
+  directory and make sure all pipelines still works.
+- Move `docs` folder to `cluster` folder and make sure all doc links works.
+- Move `test` folder to `cluster` folder and make sure all tests and pipelines
+  still works.
+- Move `tools` directory to `cluster\build` folder and make sure existing
```

Comment:
There are `spk` related items in `\tools\prereqs`. We will have to surgically move items from `\tools\*` to `cluster\build`